jobs/build-node-image.Jenkinsfile

```diff
@@ -65,67 +68,48 @@ lock(resource: "build-node-image") {
 
         stage('Build Layered Image') {
             withCredentials([file(credentialsId: 'oscontainer-push-registry-secret', variable: 'REGISTRY_SECRET')]) {
-                def build_from = params.FROM ?: stream_info.from
-                parallel arches.collectEntries{arch -> [arch, {
-                    pipeutils.withPodmanRemoteArchBuilder(arch: arch) {
-                        shwrap("""
-                            cosa remote-build-container --arch $arch \
-                                --git-ref $commit --force \
-                                --git-url ${src_config_url} \
-                                --repo ${container_registry_staging_repo} \
-                                --push-to-registry --auth=\${REGISTRY_SECRET} \
-                                --secret id=yumrepos,src=\$(pwd)/yumrepos/${stream_info.yumrepo.file} \
-                                --mount-host-ca-certs \
-                                --security-opt label=disable \
-                                --from ${build_from} \
-                                --tag ${params.RELEASE}-${arch}-${shortcommit}
-                        """)
-                    }
-                }]}
+                 def build_from = params.FROM ?: stream_info.from
+                 pipeutils.build_and_push_image(arches: arches,
+                                                src_commit: commit,
+                                                src_url: src_config_url,
+                                                staging_repository: container_registry_staging_repo,
+                                                image_tag_staging: container_registry_staging_image_tag,
+                                                manifest_tag_staging: container_registry_staging_manifest_tag,
+                                                registry_secret_path: REGISTRY_SECRET,
```

Comment:
Minor/optional: does it not work to just change `REGISTRY_SECRET` above to `REGISTRY_AUTH_FILE` and then drop this?